
# [CVE-2014-1928](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2014-1928)

## Description

- `The shell_quote function in python-gnupg 0.3.5 does not properly escape characters, which allows context-dependent attackers to execute arbitrary code via shell metacharacters in unspecified vectors, as demonstrated using "\" (backslash) characters to form multi-command sequences, a different vulnerability than CVE-2014-1927.  NOTE: this vulnerability exists because of an incomplete fix for CVE-2013-7323.`

## Cvss Data

- **Access Vector**:
  - LOCAL
- **Base Score**:
  - 4.6

## Scores

- **Exploitability Score**:
  - 3.9
- **Impact Score**:
  - 6.4
- **Base Severity**:
  - MEDIUM

## Other Information

- **Publish Date**:
  - 2014-10-25 21:55:03
- **Vulnerability Status**:
  - Analyzed

## References

- **cve@mitre.org**: http://seclists.org/oss-sec/2014/q1/246
- **cve@mitre.org**: http://seclists.org/oss-sec/2014/q1/294
- **cve@mitre.org**: http://secunia.com/advisories/56616
- **cve@mitre.org**: http://secunia.com/advisories/59031
- **cve@mitre.org**: http://www.debian.org/security/2014/dsa-2946
- **cve@mitre.org**: https://code.google.com/p/python-gnupg/
- **cve@mitre.org**: https://code.google.com/p/python-gnupg/issues/detail?id=98
